lib/build.gradle

```diff
@@ -105,10 +105,16 @@ dependencies {
 	flexmarkCompileOnly 'com.vladsch.flexmark:flexmark-all:0.62.2'
 }
 
+configurations.named('testCompatKtLint0Dot48Dot0Implementation').configure {
```

Comment:
This should no longer be necessary with version 0.4.0 of the version-compatibility-plugin, if the `compatKtLint0Dot48Dot0CompileOnly` is changed to `ompatKtLint0Dot48Dot0CompileAndTestOnly`.

Reply:
I've used the latest plugin to clean this up. Thanks for the new configuration!

Reply:
No problem, I'm happy to improve the plugin :) I also improved the plugin documentation, fyi.